README.md

```diff
@@ -166,7 +167,7 @@ zone.  This role has a trust relationship with the users account.
 | [terraform_remote_state.pca_staging](https://registry.terraform.io/providers/hashicorp/terraform/latest/docs/data-sources/remote_state) | data source |
 | [terraform_remote_state.terraform](https://registry.terraform.io/providers/hashicorp/terraform/latest/docs/data-sources/remote_state) | data source |
 
-## Inputs ##
+## Inputs
```

Comment:
Revert this change that terraform-docs makes:
```suggestion
## Inputs ##
```

Reply:
Fixed in https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/103/commits/5de085cd1da6a42095891d88144f4bf0b14543a4.